ReadMe.rst

```diff
@@ -15,61 +15,48 @@ For more details about the repository, refer to `RepoDetails.md`_.
 
 .. _`RepoDetails.md`: https://github.com/microsoft/mu_basecore/blob/HEAD/RepoDetails.md
 
-Branch Status - release/202311
+Branch Status - release/202405
 ==============================
 
 :Status:
   In Development
 
 :Entered Development:
-  2023/12/15
+  2023/11/24
 
 :Anticipated Stabilization:
-  Feb 2024
+  Nov 2024
 
-Branch Changes - release/202311
+Branch Changes - release/202405
 ===============================
 
+202405 is a larger deviation than previous releases. As part of upstreaming changes to EDK2, the commits were reviewed, squashed, and some were dropped.
+Due to these changes, there maybe more work minor work required to bring an existing platforms up to 202405 compatibality. 
+
 Breaking Changes-dev
 --------------------
-
-- Incomplete
+- SourceLevelDebugPkg has been dropped. Please reevaluate useage of this Package.
+- MemoryBinOverrideLib and associated changes have been deprecated and dropped. Platforms should remove this library instance. 
+- MdeModulePkg/Application/MpServicesTest/MpServicesTest.inf has been dropped. Any reliance on this should move to using mu_plus's UefiTestingPkg/FunctionalSystemTests/MpManagement.
+- NetworkPkg/SharedNetworkingPkg and NetworkPkg/SharedNetworkingPkg.fdf.inc have been dropped. Please include specific inf files in projects and directly build.
+- PerformancePkg has been dropped. The functionality will be available under MsApplicationPkg in mu_plus.

```

Comment:
@spbrogan, I think you mentioned you wanted this in `MsCorePkg`?

Reply:
It's still in PR here https://github.com/microsoft/mu_plus/pull/509.

Reply:
Updated to reference UefiTestingPkg.